.github/workflows/ci.yml

```diff
@@ -0,0 +1,56 @@
+name: CI
+
+on: [
+    push,
+    pull_request
+]
+
+jobs:
+  validation:
+    runs-on: ubuntu-18.04
+    strategy:
+      fail-fast: false
+      matrix:
+        php-version: [
+            '7.4',
```

Comment:
Wait, this is missing our minimum, 7.3
otherwise we are blind on current min spec